CHANGELOG.md

```diff
@@ -1,9 +1,18 @@
-# dbt_linkedin v0.UPDATE.UPDATE
-
- ## Under the Hood:
-
+# dbt_linkedin v0.8.0
+[PR #32](https://github.com/fivetran/dbt_linkedin/pull/32) includes the following updates:
+## Feature update 🎉
+- Unioning capability! This adds the ability to union source data from multiple linkedin connectors. Refer to the [README](https://github.com/fivetran/dbt_linkedin/blob/main/README.md) for more details.
```

Comment:
Actually, can we remove the README "hide" sections in these updates. This way we are able to link directly to the section in the README where the users needs to understand how to configure the union feature.

Otherwise, I worry this cool feature is hidden from people actually using it.

Reply:
Updated.